processes/ingest/hathi_trust.py

```diff
@@ -36,45 +37,50 @@ def runProcess(self):
 
         logger.info(f'Ingested {len(self.records)} Hathi Trust records')
 
-    def importRemoteRecords(self, fullOrPartial=False):
-        self.importFromHathiTrustDataFile(fullDump=fullOrPartial)
-
-    def importFromSpecificFile(self, filePath):
+    def importFromSpecificFile(self, file_path):
         try:
-            hathiFile = open(filePath, newline='')
+            hathi_file = open(file_path, new_line='')
         except FileNotFoundError:
             raise IOError('Unable to open local CSV file')
 
-        hathiReader = csv.reader(hathiFile, delimiter='\t')
+        hathiReader = csv.reader(hathi_file, delimiter='\t')
         self.readHathiFile(hathiReader)
 
-    def parseHathiDataRow(self, dataRow):
-        hathiRec = HathiMapping(dataRow, self.statics)
+    def parseHathiDataRow(self, data_row):
+        hathiRec = HathiMapping(data_row, self.statics)
         hathiRec.applyMapping()
         self.addDCDWToUpdateList(hathiRec)
 
 
-    def importFromHathiTrustDataFile(self, fullDump=False):
+    def importFromHathiTrustDataFile(self, full_dump=False, start_date_time=None):
         try:
-            fileList = requests.get(os.environ['HATHI_DATAFILES'], timeout=15)
-            fileList.raise_for_status()
+            file_list = requests.get(os.environ['HATHI_DATAFILES'], timeout=15)
+            file_list.raise_for_status()
         except (ReadTimeout, HTTPError):
             raise IOError('Unable to load data files')
 
-        fileJSON = fileList.json()
+        file_json = file_list.json()
 
-        fileJSON.sort(
+        file_json.sort(
             key=lambda x: datetime.strptime(
                 x['created'],
                 self.returnHathiDateFormat(x['created'])
             ).timestamp(),
             reverse=True
         )
 
-        for hathiFile in fileJSON:
-            if hathiFile['full'] == fullDump:
-                self.importFromHathiFile(hathiFile['url'])
+        for hathi_file in file_json:
+            if not hathi_file.get('full') and not full_dump:
+                hathi_date_format = self.returnHathiDateFormat(hathi_file.get('modified'))
+                hathi_date_modified = datetime.strptime(hathi_file.get('modified'), hathi_date_format).replace(tzinfo=None)
+                if start_date_time and hathi_date_modified > start_date_time:
+                    self.importFromHathiFile(hathi_file.get('url'), start_date_time)
+                    break       
+            elif hathi_file.get('full') and full_dump:
+                self.importFromHathiFile(hathi_file.get('url'))
                 break
```

Comment:
Do we want to break here or do we want to import the file and move on to the next file?

Reply:
I kept the break because the original code here had a break after we called importFromHathiFile which I'm only assuming was put here so only one hathifile would be imported from all the hathifiles since they could be so massive. I think it's fine to remove them since we will only ingest hathi data rows that were updated in the past day so there won't be as much hathi rows ingested in general. What are your thoughts?

```
if hathiFile['full'] == fullDump:
                        self.importFromHathiFile(hathiFile['url'])
                        break
```